docs/source/distributions.rst

```diff
@@ -1,8 +1,250 @@
-Distributions
-=============
+Base Distribution
+=================
 
-.. automodule:: numpyro.distributions
+Distribution
+------------
+.. autoclass:: numpyro.distributions.Distribution
     :members:
     :undoc-members:
     :show-inheritance:
     :member-order: bysource
+
+TransformedDistribution
+-----------------------
+.. autoclass:: numpyro.distributions.TransformedDistribution
+    :members:
+    :undoc-members:
+    :show-inheritance:
+    :member-order: bysource
+
+
+Continuous Distributions
+========================
+
+Beta
+----
+.. autoclass:: numpyro.distributions.continuous.Beta
+    :members:
+    :undoc-members:
+    :show-inheritance:
+    :member-order: bysource
+
+Cauchy
+------
+.. autoclass:: numpyro.distributions.continuous.Cauchy
+    :members:
+    :undoc-members:
+    :show-inheritance:
+    :member-order: bysource
+
+Chi2
+----
+.. autoclass:: numpyro.distributions.continuous.Chi2
+    :members:
+    :undoc-members:
+    :show-inheritance:
+    :member-order: bysource
+
+Dirichlet
+---------
```

Comment:
This was needed so that the sidebar is nicely populated, with just `automodule` that doesn't work. Also this helps us control the layout, e.g. show Distributions followed by X..